test/integration/tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
```

Comment:
Would be happy to do that. I'm not an expert on tox config :) but it looked to me like that would mean specifying ansible version individually in tox.ini on each environment. Otherwise, if I left the ansible requirement in shared requirements.txt and tried to specify a different one just for integration, it complained about the mismatch. Is there a way to do it?